src/Models/Backup.php

```diff
@@ -187,6 +187,7 @@ public function serialize()
             'expiry'    => $this->getExpiry(),
             'initiator' => $this->getInitiator(),
             'url'       => $this->get('archive_url'),
+            'type'      => $this->get('type'),
```

Comment:
We'll want to make sure this gets covered in unit testing. We'll look at this together.

Reply:
Ok.